portal-api/src/scheduler/types/scheduler-job.d.ts

```diff
@@ -9,7 +9,7 @@ export type SchedulerJob = {
      */
     message: string;
     /**
-     * The task to be run by the job
+     * The task to be run by the jobs
```

Comment:
This comment is in reference to this type, being a single job, so I don't think this should be plural?